spec/like_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe 'creation' do
+    before do
+      @user = User.create(name: 'John Doe', photo: 'https://picsum.photos/200/300', bio: 'I am John Doe')
+      @post = Post.create(author_id: @user.id, title: 'My Post', text: 'This is my post')
+      @like = Like.create(author_id: @user.id, post_id: @post.id)
+    end
+
+    it "user_id can't be blank" do
+      @like.author_id = nil
+      expect(@like).to_not be_valid
+    end
+
+    it 'user_id must be an integer' do
+      @like.author_id = 'a'
+      expect(@like).to_not be_valid
+    end
+
+    it 'user_id must be greater than 0' do
+      expect(@like).to_not be_valid
+    end
+
+    it "post_id can't be blank" do
+      @like.post_id = nil
+      expect(@like).to_not be_valid
+    end
+
+    it 'post_id must be an integer' do
+      @like.post_id = 'a'
+      expect(@like).to_not be_valid
+    end
+
+    it 'post_id must be greater than 0' do
+      expect(@like).to_not be_valid
+    end
+  end
+end
```

Comment:
- Some blocks here are also failing due to the same reason I mentioned in the `comment` spec file. Alternatively, the `before :all` method will also fix the issue. Please fix this 🙏 

![gjcgcgcg](https://user-images.githubusercontent.com/73607512/236181860-f6df161f-d8d0-494d-99e8-b2b0419b6a3b.PNG)

Reply:
Fixed.